### DISCUSSION TITLE
Get organization list with member using api

### DISCUSSION THREAD

**Discussion by gokulraghuthaman** [zitadel/zitadel#7098] [Q&A]
Hi,
I am trying to create an organization-user related project using zitadel and the hierarchy is the same as in zitadel. Now I am stuck with the apis, the scenario is I need to list the organizations which is I am a member of. for example if I am a manager of 5 organizations with different roles i need to list those 5 organizations only. But I looked into the api docuentation but as an instance level user, listing of all organizations api only there. Is there any other way to accomplish this using zetadel apis?

**Comment by hifabienne** [comment]
Hei there are different possibilities.
If you want to get a list of organizations where the user has a user grant you can get this request: https://zitadel.com/docs/apis/resources/auth/auth-service-list-my-project-orgs

If you want to get a list of the permissions a user has within ZITADEL you can use this one: https://zitadel.com/docs/apis/resources/auth/auth-service-list-my-zitadel-permissions

Not that this is the authentication API, it returns the result for the current authenticated user.

**Reply by gokulraghuthaman** [reply]
But I am not creating any relations with project here, is there any other organization listing apis are there with user?

**Reply by hifabienne** [reply]
With the second call there are no relations to projects. Its only about memberships of the user.
